libs/api/domains/sessions/src/lib/dto/paginated-session.response.ts

```diff
@@ -0,0 +1,8 @@
+import { ObjectType } from '@nestjs/graphql'
+
+import { PaginatedResponse } from '@island.is/nest/pagination'
+
+import { Session } from '../models/session.model'
+
+@ObjectType('SessionsPaginatedSessionResponse')
```

Comment:
Our graphql naming convention is ... tricky 🙃

Reply:
Yes the `Session` model is best... `SessionsSession` 😅